src/App.tsx

```diff
@@ -10,9 +12,18 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/" element={<HomePage />} />
+            <Route
+              path="*"
+              element={<h1 className="title">Page not found</h1>}
+            />
+
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path=":personHref" element={<PeoplePage />} />
```

Comment:
The `PeoplePage` component is used for both the parent and child routes. This might not be the intended behavior and could lead to unexpected results. Consider using a different component for the child route or refactoring `PeoplePage` to handle both scenarios appropriately.